common/Configuration.py

```diff
@@ -1,24 +1,141 @@
+import os
+import shutil
+from pathlib import Path
+
+from ._fill_in_template import fill_in_template_file
+
+
 class Configuration(object):
-    def __init__(self):
+    """
+    Class representing a Frescox input configuration.
+    """
```

Comment:
FYI, the Sphinx docs are likely configured to use the `__init__`'s docstring as the main documentation for the class.  Therefore, at present this docstring is probably not rendered.  This sentence could be moved down to `__init__`'s docstring.

With this configuration, the class's main description is rendered for the class's constructor as `bfrescox.Configuration` rather than `bfrescox.__init__` so that the presentation is compact and nice.